handlers.go

```diff
@@ -179,6 +217,14 @@ func (b *BouncerHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	product = strings.TrimSpace(strings.ToLower(product))
 	os = strings.TrimSpace(strings.ToLower(os))
 
+	// HACKS
+	// If the user is coming from windows xp, send a sha1
+	// signed product.
+	// HACKS
+	if isWindowsXPUserAgent(req.UserAgent()) {
```

Comment:
However, this also prevents older versions of Firefox from being downloaded, which is [breaking tests](https://travis-ci.org/mozilla-services/go-bouncer/jobs/98148919) and quite possibly a great number of use cases, such as building a VM with an older version of windows and Firefox for compatibility testing.